### ISSUE TITLE
[feature request] more settings for "map while navigation"

### ISSUE THREAD

**Issue by SnejPro** [osmandapp/OsmAnd#19968]
### Describe the idea (required)

An option in the settings to set the map orientation and 3D mode only during navigation.

### Tell us about the expected behaviour (required)

I have a specific profile for my professional needs (e.g. all hospital POIs enabled, custom routing.xml).
I'd like to "browse" the map in 2D and map north-orientated. But when as soon as I starte the navigation, I'd like the map to switch automatically in 3D and movement-oriented mode.

This could be achieved in the settings section "Map while navigation".
![Screenshot_20240528-123422](https://github.com/osmandapp/OsmAnd/assets/11321022/560d1449-57e9-4a86-9883-2c5999df8bee)

### Tell us about alternatives you've considered (required)

Using of different profiles, but then you have to make all settings twice.

**Comment by Vucumpra** [comment]
Yes, great idea, a new item in setup to choose if map switch automatically from 2D to 3D when starting navigation.
It will be nice if this new option will be configurable by profile and not for all OsmAnd
